addons/pos_gift_card/static/src/js/GiftCardPopup.js

```diff
@@ -65,7 +66,7 @@ odoo.define("pos_gift_card.GiftCardPopup", function (require) {
             fields: ["code", "initial_amount", "balance"],
           });
           if (giftCard.length) {
-            this.env.pos.giftCard.push(giftCard[0])
+            this.env.pos.giftCard.push(giftCard[0]);
```

Comment:
Don't add a missing semi-colon on a line that is not part of your fix.
It will add unnecessary diff.